lib/quby/answers/services/updates_answers.rb

```diff
@@ -49,6 +46,20 @@ def on_failure(&block)
 
         protected
 
+        def save_raw_params_on_error(new_attributes, &_block)
```

Comment:
Er gebeurt een boel met de state van answer in de update, het punt is juist dat ik die rare state niet wil meenemen bij het opslaan van answer in het geval van errors. Het doel van de functie is raw params opslaan in geval van excepties, en dat is ook precies wat ie doet. Het opslaan in het geval van geen excepties is een concern van wat er in de yield gebeurt.

 Ensure wordt ook in geval van geen exceptie aangeroepen, terwijl het opslaan in de huidige code juist al in de yield gebeurt. In een ensure block testen of je daarvoor toch net wel een exceptie hebt gehad is nogal zinloos, daar is het rescue block voor.